docs/operator-guide/gebco.md

```diff
@@ -0,0 +1,49 @@
+# Gebco re-project to NZTM2000Quad cog file
+
+[GEBCO](https://www.gebco.net/) grid data are based on WGS84 which required to re-project into NZTM2000Quad tilematrix before importing into Basemaps. This instruction include the process of re-project [Gebco data](https://www.gebco.net/data_and_products/gridded_bathymetry_data/) into NZTM2000Quad tiff file by using gdal commands. Then we can process the output by [cogify](https://github.com/linz/basemaps/tree/master/packages/cogify) commands to import into LINZ Basemaps.
+
+## Process
+
+Download the lastest [Gebco gridded bathymetry data](https://www.gebco.net/data_and_products/gridded_bathymetry_data/).
+
+Build VRT first with the Gdal command.
+
+```bash
+docker run
+    --rm -it -v $PWD:$PWD
+    --workdir $PWD ghcr.io/osgeo/gdal:ubuntu-small-3.8.3
+    gdalbuildvrt
+    geotiff_output/gebco_2023.vrt
+    gebco_2023_geotiff/*.tif
+```
+
+Wrap VRT and re-project into 2193 projection
+
+```bash
+docker run
+    --rm -it -v $PWD:$PWD
+    --workdir $PWD ghcr.io/osgeo/gdal:ubuntu-small-3.8.3
+    gdalwarp
+    -of VRT
+    -multi
```

Comment:
We generally add a `-r bilinear` for most resampling here, Im not sure what the default is.

It would be also good to warp it to float32